src/types/redis_sb_chain.cc

```diff
@@ -0,0 +1,240 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+#include "redis_sb_chain.h"
+
+namespace redis {
+
+void SBChain::appendBFSuffix(const Slice &ns_key, uint16_t filters_index, std::string *output) {
+  output->clear();
+
+  output->append(ns_key.data(), ns_key.size());
+  output->append(kBloomFilterSeparator);
+  PutFixed16(output, filters_index);
+}
+
+void SBChain ::appendBFMetaSuffix(const Slice &bf_key, std::string *output) {
+  output->clear();
+
+  output->append(bf_key.data(), bf_key.size());
+  output->append(kBloomFilterSeparator);
+  output->append("meta");
+}
```

Comment:
Could these field be encoded as subkeys? So that the compaction procedure can work well.

Refer to the key encoding https://kvrocks.apache.org/community/data-structure-on-rocksdb#user-key-encoding

Reply:
Sure, these shoule be encoded as subkeys. I will fix it.